az-core/src/main/java/azkaban/utils/JSONUtils.java

```diff
@@ -271,4 +274,20 @@ private static boolean isCharSpecialUnicode(final char ch) {
 
     return false;
   }
+
+  /**
+   * Reads json file from the classpath placed in the resources folder and returns as string
+   * @param filePath
+   * @return String json as string
+   */
+  public static String readJsonFileAsString(String filePath) {
+    try {
+      InputStream is = JSONUtils.class.getClassLoader().getResourceAsStream(filePath);
+      return IOUtils.toString(is, StandardCharsets.UTF_8);
+    } catch (IOException e) {
```

Comment:
Can you please use catch (IOException|Exception)?

Reply:
Multi catch exception must be disjoint. IOException is sub class of Exception and hence we can't have them together.